packages/kuma-gui/src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,7 +1,7 @@
 zone-cps:
   docs: &zone-cps.docs
     type: docs
-    label: Documentation
+    label: Learn more
```

Comment:
I think this change needs to be applied in `meshes` also

Reply:
updated